src/Stryker.Core/Stryker.Core/Initialisation/InputFileResolver.cs

```diff
@@ -137,11 +138,17 @@ private void MarkInputFilesAsExcluded(FolderComposite root, List<string> pathsTo
 
         public string ScanProjectFile(string currentDirectory)
         {
-            var projectFiles = _fileSystem.Directory.GetFiles(currentDirectory, "*.csproj", SearchOption.AllDirectories);
+            var projectFiles = _fileSystem.Directory.GetFiles(currentDirectory, "*.csproj");
             _logger.LogTrace("Scanned the current directory for *.csproj files: found {0}", projectFiles);
             if (projectFiles.Count() > 1)
             {
-                throw new StrykerInputException("Expected exactly one .csproj file, found more than one. Please fix your project contents");
+                var sb = new StringBuilder();
+                sb.AppendLine("Expected exactly one .csproj file, found more than one:");
+                foreach (var file in projectFiles)
```

Comment:
Please add the braces for this foreach and i'll merge the pr

Reply:
Done, thanks for your comment.